AlphaWallet/UI/Views/AmountTextField.swift

```diff
@@ -81,7 +82,12 @@ class AmountTextField: UIControl {
     }()
 
     init() {
-        currentPair = Pair(left: .eth, right: .usd)
+        let chainId = Config().chainID
+        if chainId == 100 {
```

Comment:
I think it's better to use a switch right away. Much easier to discover this piece of code rather than remember to search for 100 later.

Can you help look for this line in `AmountTextField.swift` and see if it's broken? I'm already regretting not using a switch here, otherwise, you'd found it. (and search for explicit references to `.eth` and `.usd` in the same file for similar bugs..

```
if currentPair.left != .eth {
```